File: packages/taiyaki-aot-compiler/benchmarks/bench_array_sum.py
def array_sum(n):
    arr = []
    i = 0
    while i < n:
        arr.append(i)
        i += 1
    s = 0
    j = 0
    while j < n:
        s = s + arr[j]
        j += 1
    return s

result = array_sum(1000000)
print(result)

File: packages/taiyaki-aot-compiler/benchmarks/bench_array_sum.js
function arraySum(n) {
  const arr = [];
  let i = 0;
  while (i < n) {
    arr.push(i);
    i = i + 1;
  }
  let sum = 0;
  let j = 0;
  while (j < n) {
    sum = sum + arr[j];
    j = j + 1;
  }
  return sum;
}

console.log(arraySum(1000000));
